src/cli.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+
+import fs from 'fs'
+import yargs from 'yargs'
+
+import CONSTANTS from './constants'
+import getApiCredentials from './get-api-credentials'
+import CsvParserPrice from './main'
+
+process.title = 'csvparserprice'
+
+const args = yargs
+  .usage(
+    `\n
+Usage: $0 [options]
+Convert commercetools price CSV data to JSON.`
+  )
+  .showHelpOnFail(false)
+
+  .option('help', {
+    alias: 'h',
+  })
+  .help('help', 'Show help text.')
+
+  .option('inputFile', {
+    alias: 'i',
+    default: 'stdin',
+    describe: 'Path to CSV file.',
+  })
+  .coerce('inputFile', (arg) => {
+    if (arg !== 'stdin')
+      return fs.createReadStream(String(arg))
+
+    return process.stdin
+  })
+
+  .option('outputFile', {
+    alias: 'o',
+    default: 'stdout',
+    describe: 'Input CSV file.',
+  })
+  .coerce('outputFile', (arg) => {
+    if (arg !== 'stdout')
+      return fs.createWriteStream(String(arg))
+
+    return process.stdout
+  })
+
+  .option('batchSize', {
+    alias: 'b',
+    default: CONSTANTS.standardOption.batchSize,
+    describe: 'Number of CSV rows to handle simultaneously.',
+  })
+
+  .option('delimiter', {
+    alias: 'd',
+    default: CONSTANTS.standardOption.delimiter,
+    describe: 'Used CSV delimiter.',
+  })
+
+  .option('strictMode', {
+    alias: 's',
+    default: CONSTANTS.standardOption.strictMode,
+    describe: 'Parse CSV strictly.',
+  })
+
+  .option('projectKey', {
+    alias: 'p',
+    describe: 'API project key.',
+    demand: true,
+  })
+
+  .option('host', {
+    describe: 'HTTP client host parameter.',
+  })
+
+  .option('protocol', {
+    describe: 'HTTP client protocol parameter.',
+  })
+
+  .option('accessToken', {
+    describe: 'HTTP client access token.',
+  })
+  .argv
+
+// Handle an error by logging and exiting the process
+args.outputFile
+  .on('error', (error) => {
+    process.stderr.write(error.message)
+    process.exit(1)
+  })
+
+getApiCredentials(args.projectKey, args.accessToken)
+  .then(apiCredentials =>
+    new CsvParserPrice(
+      {
+        debug: process.stdout,
+        error: process.stderr,
+        info: process.stdout,
+        trace: process.stdout,
+      },
+      {
+        config: apiCredentials,
```

Comment:
@Siilwyn we should pass the `user_agent` in the config. something like [this](https://github.com/sphereio/sphere-stock-import/blob/master/src/coffee/stockimport.coffee#L16)